include/graph_configuration.h

```diff
@@ -30,6 +30,12 @@ class GraphConfiguration {
   // How many OMP threads each graph worker uses
   size_t _group_size = 1;
 
+  // Option to create more sketches than for standard connected components
+  // Ex factor of 1, double the sketches
+  //    factor of 0.5, 1.5 times the sketches
+  //    factor of 0, normal quantity of sketches
+  double _adtl_skts_factor = 0;
```

Comment:
my engineer brain dislikes this very much. would be simpler for it to be just multiplicative and name it accordingly

Reply:
Hmm okay I suppose that works at well.

Reply:
Done